controllers/admin/AdminAccessController.php

```diff
@@ -223,4 +226,45 @@ private function getChildrenTab(array &$tabs, $id_parent = 0)
 
         return $children;
     }
+
+    /**
+     * @param int $tab_id
+     * @param string $permission
+     * @param int $profile_id
+     *
+     * @return bool
+     *
+     * @throws Exception
+     */
+    private function isAllowedToEditPermissions($tab_id, $permission, $profile_id)
+    {
+        if ($tab_id === -1) {
+            return false;
+        }
+
+        $currentProfile = (int) ContextCore::getContext()->employee->id_profile;
+
+        /**
+         * The current employee can't change its own permissions
+         */
+        if ($profile_id === $currentProfile) {
```

Comment:
so an admin or super admin user can't even edit his own permissions?

Reply:
yes, it's the idea!